srcpkgs/pmw/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'pmw'
+pkgname=pmw
+version=5.20
+revision=1
+build_style=gnu-configure
+make_install_args="MANDIR=\${DESTDIR}/usr/share/man"
+short_desc="Program for high quality music typesetting"
+maintainer="Stanislav Paskalev <spaskalev@protonmail.com>"
+license="GPL-2.0-or-later"
+homepage="http://quercite.dx.am/pmw.html"
+changelog="https://raw.githubusercontent.com/PhilipHazel/pmw/main/doc/ChangeLog"
+distfiles="https://github.com/PhilipHazel/pmw/releases/download/pmw-${version}/pmw-${version}.tar.gz"
+checksum=a78063bcd59b72ad31e6ad471270bfc0d28db38ecb74cec20e06503ad033c8b4
+
+post_install() {
+	vinstall doc/spec.pdf 644 /usr/share/pmw manual.pdf
```

Comment:
use vdoc, unless this is where the application looks for this (if it does so)

Reply:
Done (this is a user manual, the application does not depend on it)